README.md

```diff
@@ -33,7 +33,11 @@ node app
 
 ## Run
 
-Refresh the DataLake views. The views give fast access to enrollment and assignment data.
+Refresh the DataLake views. The materialized views are intended to give fast access to analytics data to the application layer.
 
-1. Configure the cron-like job. The `dataLake.cronJob.cronTime` config must have a [valid cron pattern](http://crontab.org).
+1. Configure the cron-like job and the script to run. The `dataLake.cron.tasks` configs must have a [valid cron pattern](http://crontab.org) and the cron script to run on trigger.
```

Comment:
@sandeepmjay The above is also a bit unclear. :)  Are you describing two separate configurations? Or, one config for two processes?  Perhaps some sample code to illustrate?